lib/cc/analyzer/engine.rb

```diff
@@ -63,6 +63,7 @@ def container_name
 
       def config_file
         path = File.join("/tmp/cc", SecureRandom.uuid)
+        FileUtils.mkdir_p("/tmp/cc")
```

Comment:
This was a change I had to make to get things working in my local dev environment (where I run everything within a Docker container). This directory seems to need to exist to write the config, and I don't see where we are creating it. This seemed safe enough, and should no-op if it is already there.